src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -311,11 +310,11 @@ Before you enable sign-up, sign-in, and sign-out there are a few decisions you w
 #### Which Amazon Cognito attributes will you pass for sign-up?
 
 User attributes (e.g. username, email, etc.) are used to identify your users. These are set up when you configure your Cognito User Pool and cannot be renamed or deleted after they are configured. You will need to know which attributes were configured as required at set up (e.g. email) and make sure your user is prompted to enter this information. You can find this information in the Amazon Cognito console under the *Sign-up experience* tab. You can also add [additional User Pool attributes]( https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html) that are collected from the sign-up form. This includes attributes like address, birthdate, phone, etc. These attributes are not required by default. You can make these additional attributes required but it cannot be changed back to not required once configured.
-
+ 
```

Comment:
Is there a way to find attributes set up during the User Pool configuration via an API call if they want to quickly check without going into the console? May be good to add to the fourth sentence above.

Reply:
Not at this time - something we are considering in a future version of Amplify - to provide some types they can reference to see what attributes are set up in their backend.